pallets/nfts/src/lib.rs

```diff
@@ -311,10 +311,12 @@ pub mod pallet {
         SeriesNotFound,
         /// No NFT was found with that NFT id.
         InvalidNFTId,
-        /// Ipfs reference is too short.
-        TooShortIpfsReference,
-        /// Ipfs reference is too long.
-        TooLongIpfsReference,
+        /// Ipfs reference length is shorter than what
+        /// is defined in MinStringLength.
```

Comment:
I would avoid mentioning anything related to some internal defined values like MinStringLength.